src/modules/menu/components/activeLayers/MenuActiveLayersListItem.vue

```diff
@@ -204,7 +204,7 @@ function duplicateLayer() {
                 min="0.0"
                 max="1.0"
                 step="0.01"
-                :value="layer.opacity"
+                :value="1.0 - layer.opacity"
                 :data-cy="`slider-opacity-layer-${id}-${index}`"
                 @change="onOpacityChange"
```

Comment:
See comment above. Also while testing I noticed that a small margin before and also maybe after the slider would be great, could you add it ?
![image](https://github.com/geoadmin/web-mapviewer/assets/67745584/86d8fc91-572d-4fc2-8b07-1e619f5973dd)

Reply:
Testing with `ms-2 me-4` looks better in my opinion 
![image](https://github.com/geoadmin/web-mapviewer/assets/67745584/c842b02e-a857-462b-afbe-a050804984b3)